doc(override): Clarify controller override (#16823)
symfony/symfony-docs: bundles/override.rst

If the controller is a service, see the next section on how to override it.
Otherwise, define a new route + controller with the same path associated to the
controller you want to override (and make sure that the new route is loaded

> before the bundle one and have a different name).

Reviewer: ```suggestion
before the bundle one or have a the same name).
```
?

Reviewer 2: I think both opinions are opposite ... so I need help here to decide what to do.

Maybe @stof can comment on this? Thanks!
